install.ps1

```diff
@@ -0,0 +1,43 @@
+# Installs the native manifest on windows
+# 
+
+$app = 'com.dannyvankooten.browserpass'
+
+$dirpath = Join-Path -Path $env:localappdata -ChildPath 'browserpass'
+$ff_jsonpath = Join-Path -Path $dirpath -ChildPath "$app-firefox.json"
+$chrome_jsonpath = Join-Path -Path $dirpath -ChildPath "$app-chrome.json"
+
+# Make our local directory
+new-item -type Directory -Path $dirpath -force
+
+# copy our bin to local directory
+& cp browserpass-windows64.exe $dirpath
+
+# copy the native messaging manifest
+$ffile = gc firefox-host.json
+$ffile -replace '%%replace%%', ((Join-Path -Path $dirpath -ChildPath 'browserpass-windows64.exe' | ConvertTo-json) -replace '^"|"$', "") | Out-File -Encoding UTF8 $ff_jsonpath
+
+$cfile = gc chrome-host.json
+$cfile -replace '%%replace%%', ((Join-Path -Path $dirpath -ChildPath 'browserpass-windows64.exe' | ConvertTo-json) -replace '^"|"$', "") | Out-File -Encoding UTF8 $chrome_jsonpath
+
+if ($args[0] -eq "global") {
+    Write-Host "Installing browserpass for all users"
+    # add our registry values for all users
+    New-Item -Path "hklm:\Software\Mozilla\NativeMessagingHosts" -force
+    New-Item -Path "hklm:\Software\Mozilla\NativeMessagingHosts\$app" -force
+    New-ItemProperty -Path "hklm:\Software\Mozilla\NativeMessagingHosts\$app" -Name '(Default)' -Value $ff_jsonpath -force
+
+    #New-Item -Path "hklm:\Software\Google\Chrome\NativeMessagingHosts" -force
```

Comment:
does this line need to be uncommented? I see you have the same line above for Mozilla, but you don't have them in the `else` branch.

Reply:
In my tests it was causing an error because the entry already exists. I wanted to leave it there until more broad testing could be done.

Reply:
To confirm, do you want to finish testing before it is merged? Maybe it is something @chrboe could give you a hand on?